api/allIncidents/adminIncidentsRouter.js

```diff
@@ -101,11 +101,11 @@ router.get('/incidents/:incident_id', checkIncidentExists, async (req, res) => {
 
 /**
  * @swagger
- * /{incident_id}:
- *  PUT:
+ * /adminincidents/incidents/{incident_id}:
+ *  put:
```

Comment:
@francishtknguyen Do you have time to go over how this endpoint works? On Monday the 26th perhaps?

Reply:
Yes that works for me,